charts/capsule/values.yaml

```diff
@@ -8,6 +8,7 @@ manager:
     pullPolicy: IfNotPresent
     tag: ''
   # Additional Capsule options
+  hostNetwork: false
```

Comment:
Would be great to add the reason why `hostNetwork` could be set to _true_, we could easily copy/paste the same description used [here](https://github.com/jetstack/cert-manager/blob/15b6f32071b45687b44eb097145815fc3af1b010/deploy/charts/cert-manager/values.yaml#L304-L313) or rather point directly to the GitHub issue link.